lib/evss/pciu/configuration.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module EVSS
+  module PCIU
```

Comment:
my bad, forgot about the `mock_enabled?` method in the config, I think the only change would be to namespace it to `PCIUEmail` rather than just `PCIU` to avoid confusion with the other PCIU services.

Reply:
@jkassemi @kreek - Considering we need the unique `mock_enabled?` per service, do we want to: 

1. namespace by sub-service (i.e. `PCIUAddress`, `PCIUEmail`, etc.), or
2. namespace by common parent (i.e. `PCIU`)

Reply:
@kreek - do you think the refactor of  the `PCIUAddress` service to utilize `EVSS::PCIU::Configuration` is within the scope of this ticket, or can we break that out into a separate PR that includes a full refactor of `PCIUAddress` into `PCIU`?

Reply:
@jkassemi it could be a separate PR